internal/provider/resource_integration_servicenow.go

```diff
@@ -0,0 +1,283 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-log/tflog"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"wiz.io/hashicorp/terraform-provider-wiz/internal"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/client"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/utils"
+	"wiz.io/hashicorp/terraform-provider-wiz/internal/wiz"
+)
+
+func resourceWizIntegrationServiceNow() *schema.Resource {
+	return &schema.Resource{
+		Description: "Integrations are reusable, generic connections between Wiz and third-party platforms like Slack, Google Chat, and Jira that allow data from Wiz to be passed to your preferred tool.",
+		Schema: map[string]*schema.Schema{
+			"id": {
+				Type:        schema.TypeString,
+				Description: "Identifier for this object.",
+				Computed:    true,
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Description: "The name of the integration.",
+				Required:    true,
+			},
+			"created_at": {
+				Type:        schema.TypeString,
+				Description: "Identifies the date and time when the object was created.",
+				Computed:    true,
+			},
+			"project_id": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Description: "The project this action is scoped to.",
+			},
+			"scope": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "All Resources, Restrict this Integration to global roles only",
+				Description: fmt.Sprintf(
+					"Scoping to a selected Project makes this Integration accessible only to users with global roles or Project-scoped access to the selected Project. Other users will not be able to see it, use it, or view its results. Integrations restricted to global roles cannot be seen or used by users with Project-scoped roles. \n    - Allowed values: %s",
+					utils.SliceOfStringToMDUList(
+						internal.IntegrationScope,
+					),
+				),
+				ValidateDiagFunc: validation.ToDiagFunc(
+					validation.StringInSlice(
+						internal.IntegrationScope,
+						false,
+					),
+				),
+			},
+			"servicenow_url": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "ServiceNow URL.",
+			},
+			"servicenow_username": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Email of a ServiceNow user with permissions to create tickets",
+			},
+			"servicenow_password": {
+				Type:     schema.TypeString,
```

Comment:
think `sensitive` might be good for secret type strings in the resource

Reply:
Good catch, this is still a work in progress/incomplete.  I like having the PRs out there so I can see the results of the GitHub actions.

Reply:
Acceptance test output:

```
$ TF_ACC=1 go test ./internal/acceptance/... -v -run='TestAccResourceWizIntegrationServiceNow_basic'
=== RUN   TestAccResourceWizIntegrationServiceNow_basic
--- PASS: TestAccResourceWizIntegrationServiceNow_basic (10.11s)
PASS
ok      wiz.io/hashicorp/terraform-provider-wiz/internal/acceptance     10.137s
```